.ci/install_crio.sh

```diff
@@ -64,14 +64,9 @@ then
 fi
 
 echo "Get CRI Tools"
-critools_repo="${kubernetes_sigs_org}/cri-tools"
-go get "$critools_repo" || true
-pushd "${GOPATH}/src/${critools_repo}"
-critools_version=$(grep "ENV CRICTL_COMMIT" "${GOPATH}/src/${crio_repo}/Dockerfile" | cut -d " " -f3)
-git checkout "${critools_version}"
-go install ./cmd/crictl
-sudo install "${GOPATH}/bin/crictl" /usr/bin
-popd
+crictl_version=$(get_test_version "externals.critools.version")
```

Comment:
I went through the failure on the ubuntu job and see that it cannot find the `versions.yaml` file:
```
10:15:41 ERROR: cannot find .ci/../versions.yaml
10:15:41 Failed at 67: crictl_version=$(get_test_version "externals.critools.version")
```

@saschagrunert can you please move these lines before we `pushd` into the `cri-o` directory (L42)?, so that we are still on our CI working directory and can find the `versions.yaml` file.

Reply:
Yes, thanks for pointing me to the issue :pray: